drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
Well, the syntax covers most of the API calls quite well, we just don't support some cases yet e.g. arguments used for output and this example your provided above which is possible to handle.
I would prefer to stay with this syntax, add versioning (I will add this together with other fixes after the second round of review) and describe not yet supported functionality that we want to add.